crates/ruff_linter/src/checkers/logical_lines.rs

```diff
@@ -38,7 +38,7 @@ pub(crate) fn check_logical_lines(
 ) -> Vec<Diagnostic> {
     let mut context = LogicalLinesContext::new(settings);
 
-    let mut prev_line = None;
+    let mut non_comment_prev_line = None;
```

Comment:
Would you mind reverting this unrelated change?

Reply:
This happened because at some point I tracked previous lines (including comments) in this file, sorry for the left-over.

Done in 52589e433.

Reply:
No worries, all good :)